groups/sig-cluster-lifecycle/groups.yaml

```diff
@@ -480,6 +480,22 @@ groups:
       - dlipovetsky@d2iq.com
       - neolit123@gmail.com
 
+  - email-id: sig-cluster-lifecycle-kubespray@kubernetes.io
+    name: sig-cluster-lifecycle-kubespray
+    description: |-
+      Group for Kubespray maintainers to receive CI system notifications.
+    settings:
+      WhoCanDiscoverGroup: "ANYONE_CAN_DISCOVER"
+      WhoCanViewGroup: "ANYONE_CAN_VIEW"
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      WhoCanJoin: "ANYONE_CAN_JOIN"
+      MessageModerationLevel: "MODERATE_ALL_MESSAGES"
```

Comment:
this means you have approve all messages from members or they cannot be posted.
are you sure about this?

Reply:
i think you might want to use the same settings as these for a general-purpose ML.
https://github.com/kubernetes/k8s.io/blob/b566b8324cf5995e598ba73653e654d19749f7d8/groups/sig-cluster-lifecycle/groups.yaml#L20

Reply:
Thanks @neolit123 

Thanks for your patient guidance. It has been changed.